proxy/http2/Http2ConnectionState.cc

```diff
@@ -1406,6 +1406,8 @@ Http2ConnectionState::release_stream()
 
     if (total_client_streams_count == 0) {
       if (fini_received) {
+        ua_session->clear_session_active();
```

Comment:
Other `else if` blocks in below have `clear_session_active();` call too. Let me do refactoring in another PR. Definitely, this function ( `release_stream()` ) should be renamed at least, but I'd like to focus on just fixing the issue.